pkg/components/storage.go

```diff
@@ -22,6 +22,10 @@ import (
 )
 
 func startCSIPlugin(ctx context.Context, cfg *config.Config, kubeconfigPath string) error {
+	if !cfg.Storage.IsEnabled() {
```

Comment:
If you go from enabled to disabled in the next restart, those pods will still remain running, right? The router was in the same situation and we resolved to have MicroShift remove all the router resources if it was disabled.

Reply:
It's not as easy as that unfortunately. Without the storage logic, the kubelet simply doesn't know how to manage the backend. If there are any volumes or snapshots, these become orphaned as soon as their workload is deleted or rebooted, or if the node reboots.

Reply:
Is it possible to scan all resources to see if something would be orphaned, and if not then remove the resources? Will the instructions for removing it entirely be in the docs too?

Reply:
The uninstall workflow is documented in the EP. I still have a concern or two about automating the process, as it would still require user intervention. Let's take a little extra time, perhaps an EP, to see if it's something we want.